resources/resolver.go

```diff
@@ -79,6 +79,15 @@ func (rsrcs *Resources) Cleanup() {
 	}
 }
 
+// return file
+func (rsrc *Resources) GetFile(name string) (interface{}, error) {
```

Comment:
```suggestion 
func (rsrc *Resources) GetFile[T any](name string) (T, error) {
```

Reply:
interface{} is functionally equal to the go1.18 "any", considering we already use interface{} in the codebase, I'm going to opt to not change it. When we port gpt_bpe to go1.18 in the future, we can just do a quick find-and-replace to drop in any, which would be pretty clean.